src/evo/deterministicmns.cpp

```diff
@@ -49,6 +50,7 @@ void CDeterministicMN::ToJson(UniValue& obj) const
     UniValue stateObj;
     pdmnState->ToJson(stateObj);
 
+    obj.pushKV("type", nType == MnType::HighPerformance.index ? "HighPerformance" : "Regular");
```

Comment:
I see text constant is used at least twice: here and in rpc/masternode.cpp.

Instead better to create new const char pointer inside struct such as:
```
constexpr auto Regular = CDeterministicMNType{
    .voting_weight = 1,
    .collat_amount = 1000 * COIN,
    .descr = "Regular",
};
```


and here just write `obj.pushKV("type", getMnType(nType).desc)`